test/CTestCustom.cmake

```diff
@@ -0,0 +1 @@
+set(CTEST_CUSTOM_MAXIMUM_PASSED_TEST_OUTPUT_SIZE 8192)
```

Comment:
What does this need to be in a separate file? Why not put this one line in `test/CMakeLists.txt`?

If we need to have this second file, then you should add a comment at the top of it that explains what it does and when would anyone ever need to modify it.

Reply:
My interpretation of the relevant [CMake documentation](https://cmake.org/cmake/help/v3.4/variable/CTEST_CUSTOM_MAXIMUM_PASSED_TEST_OUTPUT_SIZE.html) was that it needed to be there.
I'll test setting it in `test/CMakeLists.txt` and comment it in either case.

Reply:
Confirmed that it cannot be set in `test/CMakeLists.txt`.  I'm glad you called attention to this though because I discovered that I missed two tests that generated more than 8kB of output.  I'll update accordingly in a few moments.